frontend/tests/functional/enterprise/settings/client.test.ts

```
@@ -0,0 +1,132 @@
+		}
+
+		if (await page.locator('#driver-dummy-element').isVisible()) {
+			await page.locator('.driver-popover-close-btn').first().click();
+		}
+		await page
+			.getByTestId('logo-image')
+			.screenshot({ path: testInfo.snapshotPath('header-logo.png') });
+	});
+
+	test.use({
+		page: async ({}, use) => {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)